src/app/components/TransactionsTable/index.tsx

```diff
@@ -91,19 +119,9 @@ export default function TransactionsTable({
                           tx.state == "pending" && "animate-pulse"
                         )}
                       >
-                        {tx.title ||
-                          tx.boostagram?.message ||
-                          (type == "incoming"
-                            ? t("received")
-                            : t(
-                                tx.state === "settled"
-                                  ? "sent"
-                                  : tx.state === "pending"
-                                  ? "sending"
-                                  : tx.state === "failed"
-                                  ? "failed"
-                                  : "sent"
-                              ))}
+                        {tx.title || tx.boostagram?.message || typeStateText}
```

Comment:
Why is boostagram message taking the place of title or typeStateText? Shouldn't it be in the description?

Reply:
Ah okay, because here we don't have descriptions. Can we unify that as well? So the date appears on the side of the title and description on the bottom?

Reply:
cool resolved. 

![image](https://github.com/user-attachments/assets/b607b3da-f97c-4b1b-85e6-4b946c6a1238)